tests/test_classicalrl/test_common/test_trainer.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+import numpy as np
+import gym
+
+from jigglypuffRL import SARSA, QLearning
+from jigglypuffRL.classicalrl import Trainer
+
+
+class TestTrainer:
```

Comment:
class not needed?

Reply:
We'll add other tests in the future, so I thought it would be better to make it a class now.

Reply:
Ok